server/game/cards/events/04/shierakqiya.js

```diff
@@ -0,0 +1,48 @@
+const DrawCard = require('../../../drawcard.js');
+
+class ShierakQiya extends DrawCard {
+
+    canPlay(player, card) {
+        if (player !== this.controller || this !== card)
+            return false;
+
+        var currentChallenge = this.game.currentChallenge;
+
+        if (!currentChallenge
+            || currentChallenge.winner !== player
+            || currentChallenge.strengthDifference < 5
+            || currentChallenge.challengeType !== 'power'
+            || player.faction.kneeled)
+            return false;
+
+        return true;
+    }
+
+    play(player) {
+        if(this.controller !== player)
+            return;
+
+        this.game.promptForSelect(player, {
+            activePromptTitle: 'Select a participating character to stand',
+            waitingPromptTitle: 'Waiting for opponent to use ' + this.name,
+            cardCondition: card =>
+                card.location === 'play area'
+                && this.game.currentChallenge.isParticipating(card),
+            onSelect: (p, card) => this.onCardSelected(p, card),
+        });
+    }
+
+    onCardSelected(player, card) {
+        player.kneelCard(player.faction);
```

Comment:
Note to self; I think this is better served with a specific method, something like player.kneelFactionCard

player.kneelCard will raise stand/kneel events that we dont necessarily want to do for faction cards.